hazelcast/src/main/java/com/hazelcast/internal/networking/ChannelFactory.java

```diff
@@ -1,34 +0,0 @@
-/*
```

Comment:
The ChannelFactory is gone. In the old design the ChannelFactory was used to make either plain or SSL channels, but TLS is now done through the pipeline. So the same channel can be used for plans and SSL transport.